src/accelerate/tracking.py

```diff
@@ -151,7 +153,9 @@ def store_init_configuration(self, values: dict):
         """
         self.writer.add_hparams(values, metric_dict={})
         self.writer.flush()
-        logger.info("Stored initial configuration hyperparameters to TensorBoard")
+        with open("hparams.yaml", "w") as outfile:
```

Comment:
Perhaps it may be good to make use of the `logging_dir` to send the yaml to? Or should these be stored separately normally.

(This part):
```python
writer =  tensorboard.SummaryWriter(self.logging_dir, **kwargs)
```

Reply:
So, should I save the yaml file in the `logging_dir`?

Reply:
Yep, as this is where users would expect it to wind up landing. E.g

logging_dir/hparams.yaml

Reply:
Yeah makes sense. I have made the changes now.

Reply:
We should be careful here that `hparams.yaml` is saved separately for separate experiments. Should make sure that when you run 2 separate runs, you get 2 separate entries in the hparams dashboard of tensorboard.